pkgs/desktops/deepin/deepin-mutter/deepin-mutter.plugins-dir.patch

```diff
@@ -0,0 +1,34 @@
+From ca79d214ae9845758a57d4b77a1bc7f478b531a3 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jos=C3=A9=20Romildo=20Malaquias?= <malaquias@gmail.com>
+Date: Sat, 20 Apr 2019 00:28:47 -0300
+Subject: [PATCH] Get plugins dir from environment variable
```

Comment:
I think I read that deepin will use `deepin-mutter` if the system has low resources?

Reply:
From the [README](https://github.com/linuxdeepin/deepin-mutter/blob/master/README.md):

> Description: Base window manager component for Deepin.
> [...]
> Mutter is extensible with plugins and supports numerous visual effects. For example, deepin-wm is written as a plugin to Mutter.
> 
> This project is a fork of Mutter to avoid DDE depending on GNOME with some special hacking work, include applying some patches, using the gsettings under com.deepin.wrap.gnome instead of org.gnome, etc.

Also I have found at [ARCHITECTURAL DESIGN](https://www.deepin.org/en/developer-community/architectural-design/)

> There are two window manager: deepin-metacity and deepin-wm, the system will automaticall choose depending on the hardware performance and display driver. deepin-metacity is a 2D window manager based on metacity for low configuration computer or without installing the proper display driver. It is low resource consumption and has bad animation effects. deepin-wm is a 3D window manager based on mutter and it's also the default window manager. It is suitable for proper display driver and high configuration computer with elegant animation effects